src/index.html

```diff
@@ -2,20 +2,67 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta 
+    <meta
       name="viewport"
       content="
-        width=device-width, 
-        user-scalable=no, 
-        initial-scale=1.0, 
-        maximum-scale=1.0, 
+        width=device-width,
+        user-scalable=no,
+        initial-scale=1.0,
+        maximum-scale=1.0,
         minimum-scale=1.0
       "
     >
     <title>Product cards</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
```

Comment:
you're only importing the 500 weight, you need 500 and 400 to use in majority of texts:
<img width="497" alt="Monosnap Product cards 2022-11-08 17-43-21" src="https://user-images.githubusercontent.com/39308717/200610197-e478ab4f-124b-473f-a496-91cae38d7851.png">